tools/bin/load_test/load_test_airbyte.sh

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env bash
+
```

Comment:
it's not for every script, but I would also consider some protective shell settings if it makes sense for your workflow 

```suggestion
# Run away from anything even a little scary
set -o nounset # -u exit if a variable is not set
set -o errexit    # -f exit for any command failure
```